pkg/git/repo.go

```diff
@@ -39,6 +39,7 @@ type Repo interface {
 	ListCommits(ctx context.Context, visionRange string) ([]Commit, error)
 	GetLatestCommit(ctx context.Context) (Commit, error)
 	GetCommitHashForRev(ctx context.Context, rev string) (string, error)
+	GetCommitForRev(ctx context.Context, rev string) (Commit, error)
```

Comment:
By adding this, I think we should mark the above `GetCommitHashForRev` as deprecated and remove it later, since we can get commit hash if we have commit object provided by this new interface 👀

Reply:
thanks, I'll delete it now.
I noticed `GetCommitForRev` is called only from `pkg/git/repo_test.go`.
Deleting it does not affect production code.